pages/stats.tsx

```diff
@@ -12,7 +12,10 @@ export default function StatsPage() {
     <div className="px-4 pt-12 lg:pt-16 mx-auto max-w-6xl">
       <NextSeo title="Stats" />
       <div className="mx-auto">
-        {userId && <TrackRecord trackRecordUserId={userId} />}
+        {userId && (
+          // TODO flag in PR that I changed the width here slightly
+          <TrackRecord trackRecordUserId={userId} className="max-w-sm" />
+        )}
```

Comment:
Unrelated to the rest of the PR: I thought the width here was too narrow, I increased it from 320px to 384px. Before/after:
![Screenshot 2024-09-12 at 12 39 57](https://github.com/user-attachments/assets/d9525a47-54cf-4f02-b1cf-e40836f189cf)
![Screenshot 2024-09-12 at 12 40 58](https://github.com/user-attachments/assets/b59f5465-018b-4643-b508-54c33c324b42)

Reply:
Looks good - the `stats` page is only linked to from the bottom nav on mobile so larger screened users are unlikely to see this